pkg/graph/schema.graphql

```diff
@@ -37,6 +37,40 @@ type RequestEdge {
   node: Request!
 }
 
+"""
+CedarAuthorityToOperate represents the response from the /authorityToOperate endpoint from the CEDAR Core API.
+"""
+type CedarAuthorityToOperate {
+  cedarId: String!
+  uuid: String!
+  actualDispositionDate: Time
+  containsPersonallyIdentifiableInformation: Boolean
+  countOfTotalNonPrivilegedUserPopulation: Int
+  countOfOpenPoams: Int
+  countOfTotalPrivilegedUserPopulation: Int
+  dateAuthorizationMemoExpires: Time
+  dateAuthorizationMemoSigned: Time
+  eAuthenticationLevel: String
+  fips199OverallImpactRating: Int
+  fismaSystemAcronym: String
+  fismaSystemName: String
+  isAccessedByNonOrganizationalUsers: Boolean
+  isPiiLimitedToUserNameAndPass: Boolean
+  isProtectedHealthInformation: Boolean
+  lastActScaDate: Time
+  lastAssessmentDate: Time
+  lastContingencyPlanCompletionDate: Time
+  lastPenTestDate: Time
+  piaCompletionDate: Time
+  primaryCyberRiskAdvisor: String
+  privacySubjectMatterExpert: String
+  recoveryPointObjective: Int
```

Comment:
This (and `recoveryTimeObjective`) should probably be defined as `Float`s instead of `Int`s to match the CEDAR schema. That'd also allow implementing the resolvers, though we'd need to convert the `float32` we get from CEDAR into a `float64`.

Reply:
GQL schema changed and resolvers implemented as of https://github.com/CMSgov/easi-app/pull/1636/commits/b421c6bbac17cdbb5fb9086448d5bcce549eb523.